docs/pages/versions/unversioned/sdk/imagepicker.md

```diff
@@ -74,7 +74,7 @@ Display the system UI for choosing an image or a video from the phone's library.
 
   Option for videos:
 
-  - **exportPreset (_[ImagePicker.ExportPresets](#imagepickerexportpresets)_)** -- **Available on iOS 11+ only.** Specify preset, which will be used to compress selected video. Defaults to `ImagePicker.ExportPresets.Passthrough`.
+  - **exportPreset (_[ImagePicker.ExportPreset](#imagepickerexportpreset)_)** -- **Available on iOS 11+ only.** Specify preset, which will be used to compress selected video. Defaults to `ImagePicker.ExportPreset.Passthrough`.
```

Comment:
I don't like this new option name at all 😞 
First thing: I don't like that this sounds like method name, because it contains some verb (`Export`). (ImagePicker.ExportedPreset sounds betters, but I would rather go with `SupportedPreset` at this point 🤔, but see second thing)
Second thing: the name mismatch with the actual behaviour: `exportPreset` vs `use selected preset to compress picked video`. Method is `launchImageLibraryAsync` and you change video picking process so I would stick with the some word that somehow describes what's happening. (names coming to my mind: `videoCompressionPreset`, `pickedVideoCompressionPreset` ❓ cc @sjchmiela, @tsapeta)

Reply:
I would say that **export** is also a noun ([source](https://dictionary.cambridge.org/dictionary/english/export))

> **the activity or business of sending goods to another country in order to sell them there:**
> - **the export of sth** _The ruling banned the export of live cattle._
> - _Most of their crops are grown **for export**._
> - _**an export company/business**_

Moreover, this is the name that Apple documentation uses — https://developer.apple.com/documentation/avfoundation/avassetexportsession/export_preset_names_for_apple_devices?language=objc

Reply:
Oh! @sjchmiela, that's correct 😉 You see I didn't check what's Apple approach for this thing 😅. So at this point I' m not sure which one should we choose 😓

Reply:
After a small discussion with @tsapeta we consider `videoCompressionPreset` to be most suitable name for this parameter 😉
From one side: we need to explicitly state that this parameter affect only video, so `videoXX` and from the other side: we can go with `xxExportPreset`, but this name does not play well with method name IMO and of course it has much sense, but not in this case 😉

Reply:
Other thing is that we can omit `video` prefix but only if use `Union Type` feature of TypeScript language for this method parameters to ensure consistency of arguments and execution.

Reply:
@lukmccall `videoExportPreset` ⬅️ go with this one 😉 this is my final word here